### PR TITLE
fix: add Innovelli LZW36 and First Alert ZCOMBO-G

### PR DESCRIPTION
**Pull request by jaburges**
Inovelli Fan / light switch lzw36.json
First Alert Zwave Combo smoke alarm zcombo-g.json

added to index.json

### CODE REVIEW

File: packages/config/config/devices/0x0138/zcombo-g.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    },
+    "paramInformation": {
+        "1": {
+            "label": "SuperVision report timeout",
+            "description": "ZCOMBO will send the message over SuperVision Command Class and it will wait for the SuperVision report from the Controller for the SuperVision report timeout time.",
+            "valueSize": 2,
+            "minValue": 500,
+            "maxValue": 5000,
+            "defaultValue": 1500,
+            "unsigned": true,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0138/zcombo-g.json]: add `"unit": "s"` please

File: packages/config/config/devices/0x0138/zcombo-g.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "3": {
+            "label": "SuperVision wait time",
+            "description": "Before retrying the message, ZCOMBO will wait for the SuperVision wait time. Actual wait time is calculated using the formula: Wait Time = SuperVision wait time base-value + random-value + (attempt-count x 5 seconds). The random value will be between 100 and 1100 milliseconds.",
+            "valueSize": 2,
+            "minValue": 1,
+            "maxValue": 60,
+            "defaultValue": 5,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0138/zcombo-g.json]: add `"unit": "s"` (is it seconds?) please

File: packages/config/config/devices/0x031e/lzw36.json [hunk 3]
```diff
@@ -0,0 +1,559 @@
...
+            "label": "Basic_Set_Fan",
+            "maxNodes": 5
+        },
+        "6": {
+            "label": "Switch_MultiLevel_Set_Fan",
+            "maxNodes": 5
+        },
+        "7": {
+            "label": "Switch_MultiLevel_Start/Stop_Fan",
+            "maxNodes": 5
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw36.json]: Replace the underscores with spaces please

File: packages/config/config/devices/0x031e/lzw36.json [hunk 4]
```diff
@@ -0,0 +1,559 @@
...
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "10": {
+            "label": "Auto Off Light Timer",
+            "description": "Automatically turns the light switch off after this many seconds. When the switch is turned on a timer is started that is the duration of this setting. When the timer expires, the switch is turned off.\n      Range:0-32767\n      Default: 0",
+            "valueSize": 2,
+            "minValue": 0,
+            "maxValue": 32767,
+            "defaultValue": 0,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw36.json]: add `"unit": "s"` please

File: packages/config/config/devices/0x031e/lzw36.json [hunk 5]
```diff
@@ -0,0 +1,559 @@
...
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "11": {
+            "label": "Auto Off Fan Timer",
+            "description": "Automatically turns the fan switch off after this many seconds. When the switch is turned on a timer is started that is the duration of this setting. When the timer expires, the switch is turned off.\n      Range:0-32767\n      Default: 0",
+            "valueSize": 2,
+            "minValue": 0,
+            "maxValue": 32767,
+            "defaultValue": 0,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw36.json]: add `"unit": "s"` please

File: packages/config/config/devices/0x031e/lzw36.json [hunk 6]
```diff
@@ -0,0 +1,559 @@
...
+            "description": "Please see website for documentation.\n          Byte 4: Effect - 0 = Off, 1 = Solid, 2 = Slow Blink, 3 = Fast Blink, 4 = Chase, 5 = Pulse\n          Byte 3: Duration - 1 to 60 = seconds, 61 to 120  minutes, 121 -  254 = hours, 255 = Indefinitely\n          Byte 2: Intensity - 0 to 9.  0 = dim, 9 = bright\n          Byte 1: Color - 0 - 255. Hue color wheel. value/255 * 360 = Hue color wheel value\n      Range: 0-100600319\n      Default: 0",
+            "valueSize": 4,
+            "minValue": 0,
+            "maxValue": 100600319,
+            "defaultValue": 0,
+            "unsigned": true,
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw36.json]: Looks like these two should be split into [partial parameters](https://zwave-js.github.io/node-zwave-js/#/development/config-files?id=partial-parameters)

File: packages/config/config/devices/0x031e/lzw36.json [hunk 7]
```diff
@@ -0,0 +1,559 @@
...
+                    "value": 8
+                },
+                {
+                    "label": "9 Seconds",
+                    "value": 9
+                },
+                {
+                    "label": "10 Seconds",
+                    "value": 10
+                }
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw36.json]: These options seem pretty unnecessary. I'd remove them and change `allowManualEntry` to true. You could mention that 0 = off in the description.

File: packages/config/config/devices/0x031e/lzw36.json [hunk 8]
```diff
@@ -0,0 +1,559 @@
...
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw36.json]: dito

File: packages/config/config/devices/0x031e/lzw36.json [hunk 9]
```diff
@@ -0,0 +1,559 @@
...
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "29": {
+            "label": "Periodic Power & Energy Reports",
+            "description": "Time period between consecutive power & energy reports being sent (in seconds). The timer is reset after each report is sent.\n      Range:0-32767\n      Default: 3600",
+            "valueSize": 2,
+            "minValue": 0,
+            "maxValue": 32767,
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw36.json]: Add `"unit": "s"`

File: packages/config/config/devices/0x031e/lzw36.json [hunk 10]
```diff
@@ -0,0 +1,559 @@
...
+            "maxValue": 100,
+            "defaultValue": 10,
+            "unsigned": true,
+            "readOnly": false,
+            "writeOnly": false,
+            "allowManualEntry": true
+        },
+        "31": {
+            "label": "Local Protection",
+            "description": "Enable local protection on these buttons. 0 = none, 1 = light, 2 = fan, 3 = both.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x031e/lzw36.json]: That should be done with selectable options

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Thanks - found a couple of things

File: packages/config/config/devices/0x0138/zcombo-g.json (hunk 1 above)
Comment (jaburges) [reply packages/config/config/devices/0x0138/zcombo-g.json]: done

File: packages/config/config/devices/0x0138/zcombo-g.json (hunk 2 above)
Comment (jaburges) [reply packages/config/config/devices/0x0138/zcombo-g.json]: done

**Review by jaburges — COMMENTED** [review]
made changes - may need you to glance over the partial parameters to ensure i've set up correctly

**Review by jaburges — COMMENTED** [review]
updated max value from failed test

**Review by jaburges — COMMENTED** [review]
valuesizes and mask for partial parameters

**Review by jaburges — COMMENTED** [review]
mask values
valuesize = 4 (replicating lzw30-sn)

**Review by jaburges — COMMENTED** [review]
updating max value of color to 255

**Review by jaburges — COMMENTED** [review]
updated default value to within max

**Comment by jaburges** [comment]
worth testing :) first

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**